src/NUnitFramework/framework/Constraints/GreaterThanOrEqualConstraint.cs

```diff
@@ -28,13 +28,30 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public class GreaterThanOrEqualConstraint : ComparisonConstraint
     {
+        /// <summary>
+        /// The Description of what this constraint tests, for
+        /// use in messages and in the ConstraintResult.
+        /// </summary>
+        private string _description;
+        
         /// <summary>
         /// Initializes a new instance of the <see cref="GreaterThanOrEqualConstraint"/> class.
         /// </summary>
         /// <param name="expected">The expected value.</param>
-        public GreaterThanOrEqualConstraint(object expected) : base(expected)
+        public GreaterThanOrEqualConstraint(object expected) : base(expected) {}
+
+        /// <summary>
+        /// The Description of what this constraint tests, for
+        /// use in messages and in the ConstraintResult.
+        /// </summary>
+        public override string Description
         {
-            Description = "greater than or equal to " + MsgUtils.FormatValue(expected);
+            get
+            {
+                if (System.String.IsNullOrEmpty(_description))
+                    _description = DefaultDescription("greater than or equal to ");
+                return _description;
```

Comment:
💡 If you wanted to, you could put `_description` in the base class (still private) and move the lazy initialization logic into `DefaultDescription`. I don't have an opinion one way or the other on this.

Reply:
Since the base concept here is overriding the `Description`, it seems, in my opinion, more clear to have corresponding field into the inheritor, rather than the base class, even though it has a smell of code duplication.